devito/dimension.py

```diff
@@ -64,12 +64,6 @@ def symbolic_end(self):
         """
         return Scalar(name=self.max_name, dtype=np.int32)
 
-    @property
-    def symbolic_extent(self):
```

Comment:
This change is now not liked by ``ir/support/space.py``, in
```
self.extent = dim.symbolic_extent + 1 + self.min_extent
```
I think we can change it to:
```
self.extent = (dim.symbolic_end - dim.symbolic_start + 1) + self.min_extent
```
The class docstring needs to change according

What do you think? I'm writing it here instead of pushing the fix because I'd like a second thought. I'm still quite convinced that the concept of ``extent`` should **not** belong to a ``Dimension``. In which case, this change would be fine. Thoughts?

Reply:
reminder?

Reply:
>  I'm still quite convinced that the concept of extent should not belong to a Dimension

Do you just mean that `extent == end - start + 1` always, so it shouldn't be overridable by a `Dimension`?  If so, I agree.

Reply:
yes. In fact, I suggest we should drop the `symbolic_extent` method from `Dimension`, if there's still one

Reply:
I think this patch already does that?

Reply:
Oh, never mind, this has been done. Cool